src/pages/index.mdx

```diff
@@ -12,35 +8,134 @@ The developer docs will teach you how to build applications, issue and use token
+| **Guide** | [Deploy an SAC](https://developers.stellar.org/docs/tokens/stellar-asset-contract) | Deploy a Stellar Asset Contract with the Stellar CLI |
+
+| **Custom Tokens** |  |  |
+| :-: | :-- | :-- |
+| **Learn** | [Asset and token introduction](https://developers.stellar.org/docs/tokens) | Learn about the difference between assets issued on Stellar and custom smart contract tokens |
+| **Learn** | [Data availability overview](https://developers.stellar.org/docs/data) | Learn about the different mechanisms for data availability on Stellar |
+| **Learn** | [Token interface (SEP-41)](https://developers.stellar.org/docs/tokens/token-interface) | Learn about the common interface for smart contract tokens on Stellar |
+| **Example** | [Token example contract](https://developers.stellar.org/docs/build/smart-contracts/example-contracts/tokens) | Demonstrates how to write a token contract that implements the Token Interface |
+| **Guide** | [Manage contracts](https://developers.stellar.org/docs/build/guides/cli/contract-lifecycle) | Use the Stellar CLI to manage smart contracts |
+| **Tool** | [Stellar CLI](https://developers.stellar.org/docs/build/guides/cli) | The command line interface to Stellar smart contracts |
+| **Tool** | [Rust SDK](https://docs.rs/soroban-sdk/latest/soroban_sdk/) | Supports writing programs for Stellar smart contracts |
+
+### Stellar for smart contract developers
+
+| **Contract Devs** |  |  |
+| :-: | --- | --- |
+| **Learn** | [Stellar smart contracts overview](https://developers.stellar.org/docs/build/smart-contracts/overview) | Learn the essentials about writing smart contracts on Stellar |
+| **Learn** | [Migrating from EVM networks](https://developers.stellar.org/docs/learn/migrate) | Stellar is not an EVM network, learn about how Stellar differs from EVM networks here |
+| **Learn** | [Storage types](https://developers.stellar.org/docs/learn/encyclopedia/storage/state-archival#contract-data-type-descriptions) | There are three different storage types for smart contract data, learn about them here |
+| **Guide** | [Getting started guide](https://developers.stellar.org/docs/build/smart-contracts/getting-started) | The go-to guide for getting setup to write smart contracts on Stellar |
+| **Guide** | [Example contracts](https://developers.stellar.org/docs/build/smart-contracts/example-contracts) | These contracts are good starting points for understanding best practices and writing your own contracts on Stellar |
+| **Guide** | [General contract guides](https://developers.stellar.org/docs/build/guides/conventions) | Various guides that highlight the norms seen in contract development |
+| **Tool** | [Stellar CLI](https://developers.stellar.org/docs/build/guides/cli) | The command line interface to Stellar smart contracts |
+| **Tool** | [Rust SDK](https://docs.rs/soroban-sdk/latest/soroban_sdk/) | Supports writing programs for Stellar smart contracts |
+
+| **Contract Devs (optional pages)** |  |  |
```

Comment:
Speaking more on readability like https://github.com/stellar/stellar-docs/pull/1005/files#r1819481542

It is hard to tell when one table ends and one begins within a single persona. 

I want to say that maybe merging the header/title columns so it is a single wide table width box would help. And also changing the background color to something more bold than light gray

Reply:
Hi @chowbao! Thank you so much for your insight and ideas on this PR. I am playing around with some formatting ideas... what do you think of the latest with dropdowns? (some of the labels in the drop-downs are not working so ignore those as I fix them, the Stellar for Applications section is working well, wdyt?)

Reply:
Oooh I like the dropdowns

It's feels more intentional/directed which I think would help with user navigation